task.sql

```diff
@@ -1 +1,39 @@
-# Write your SQL code for the database creation here. Good luck! 
+# Write your SQL code for the database creation here. Good luck!
+CREATE DATABASE ShopDB;
+
+USE ShopDB;
+
+CREATE TABLE Products (
+    ID INT AUTO_INCREMENT,
+    Name VARCHAR(50),
+    Description VARCHAR(100),
+    Price INT,
+    WarehouseAmount INT,
+    PRIMARY KEY (ID)
+);
+
+CREATE TABLE Customers (
+    ID INT AUTO_INCREMENT,
+    FirstName VARCHAR(50),
+    LastName VARCHAR(50),
+    Email VARCHAR(50),
+    Address VARCHAR(50),
+    PRIMARY KEY (ID)
+);
+
+CREATE TABLE Orders (
+    ID INT AUTO_INCREMENT,
+    CustomerID INT,
+    FOREIGN KEY (CustomerID) REFERENCES Customers (ID) ON DELETE SET NULL,
+    Date DATE,
+    PRIMARY KEY (ID)
+);
+
+CREATE TABLE OrderItems (
+    ID INT AUTO_INCREMENT,
+    OrderID INT,
+    FOREIGN KEY (OrderID) REFERENCES Orders (ID) ON DELETE SET NULL,
+    ProductID INT,
+    FOREIGN KEY (ProductID) REFERENCES Products (ID) ON DELETE SET NULL,
+    PRIMARY KEY (ID)
+);
```

Comment:
![image](https://github.com/mate-academy/rda_task_1_create_database/assets/87720583/ac3c27f6-fdeb-498e-a1eb-1786cb22fa7b)
Missed Eof(End-of-file), need to add an empty line at the end